### ISSUE TITLE
"invalid tracer config" when requesting debug_traceTransaction with callTracer

### ISSUE THREAD

**Issue by vbaranov** [paradigmxyz/reth#3765]
### Describe the bug

Reth node for ETH Mainnet returns "invalid tracer config" for most/all transactions when requesting with `callTracer` type of tracer:

A sample JSON RPC request:
```
curl -sL \
  -H "Content-Type:application/json;charset=utf-8" \
  -d "{
      \"jsonrpc\": \"2.0\",
      \"id\": 1,
      \"method\": \"debug_traceTransaction\",
      \"params\": [
          \"0x1fc829e68e4007c14ef8dedb9a39fe93f8c7907b6002d0a270068909414176b2\",
          { \"tracer\": \"callTracer\"}
      ]
  }" \
```

Response:
```
{"jsonrpc":"2.0","error":{"code":-32602,"message":"invalid tracer config"},"id":1}%
```

I am running v0.1.0-alpha.3 via docker-compose.

Docker-compose config for Reth looks like:

```
version: '3.9'

services:
  reth:
    container_name: reth
    image: ghcr.io/paradigmxyz/reth:v0.1.0-alpha.3
    restart: always
    pull_policy: always
    privileged: true
    volumes:
      - /data/reth/rethlogs:$HOME/rethlogs
      - /data/reth:$HOME/.local/share/reth
    environment:
      RUST_BACKTRACE: 'full'
    command: >
      node -vvvv
      --authrpc.jwtsecret=$HOME/.local/share/reth/mainnet/jwt.hex
      --authrpc.addr="0.0.0.0"
      --http
      --http.addr="0.0.0.0"
      --http.port="8545"
      --http.api="eth,debug,net,trace,web3,txpool"
      --ws
      --ws.addr="0.0.0.0"
      --ws.port="8546"
      --ws.api="eth,debug,net,trace,web3,txpool"
      --metrics reth:9001
      --log.directory $HOME
      --rpc-max-connections 500
    ports:
      - '9001:9001'
      - '8545:8545'
      - '8546:8546'
      - '8551:8551'

  consensus:
...
```

Is `callTracer` supported at all?

### Steps to reproduce

Send request to Reth ETH mainnet node:
```
curl -sL \
  -H "Content-Type:application/json;charset=utf-8" \
  -d "{
      \"jsonrpc\": \"2.0\",
      \"id\": 1,
      \"method\": \"debug_traceTransaction\",
      \"params\": [
          \"0x1fc829e68e4007c14ef8dedb9a39fe93f8c7907b6002d0a270068909414176b2\",
          { \"tracer\": \"callTracer\"}
      ]
  }" \
```

### Platform(s)

Linux (ARM)

### What version/commit are you on?

v0.1.0-alpha.3

### Code of Conduct

- [X] I agree to follow the Code of Conduct

**Comment by mattsse** [comment]
thanks for the repro, checking

**Comment by mattsse** [comment]
smol deserialization bug, fixed in #3766
